### PR TITLE
Prepare v0.5.0

### PR DESCRIPTION
**Pull request by stjepang**
Let's release new versions of crates.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Version 0.5.0
+
+- Update `crossbeam-channel` to 0.3.
+- Update `crossbeam-utils` to 0.6.
+- Add `AtomicCell`.
```
Comment (Vtec234) [inline CHANGELOG.md]: What about `ShardedLock` and `WaitGroup`?

File: crossbeam-deque/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Version 0.6.1
 
+- Update `crosbeam-utils` to `0.6`.
+
+# Version 0.6.1
```
Comment (Vtec234) [inline crossbeam-deque/CHANGELOG.md]: Double header?

**Comment by Vtec234** [comment]
Does this fix the two-versions-of-the-same-package issue?

**Comment by stjepang** [comment]
Thanks, fixed!

> Does this fix the two-versions-of-the-same-package issue?

Yes, we even have a check on Travis that verifies this.

**Pull request merged**